microsoft/vscode
# Markdown Static Preview Scrolling Fixes

This PR addresses two bugs with scrolling in markdown static preview:
1. When toggling between the text editor and custom (static preview) editor, this should ensure that the scrolling position stays persistent. It currently loads the editor at the top regardless of where the user scrolled last.
2. When opening a static preview in one tab and a text editor for the same file in another column, scrolling the static preview scrolls the text editor, but scrolling the text editor does not scroll the static preview.

Steps to test:
Issue 1:
- Open a markdown file that is long enough to need scrolling and scroll down.
- Use the command palette to do `View - Reopen Editor With...` > `Markdown Preview (Experimental)`
   - The resulting preview should be where you previously scrolled to in the text editor. Scroll on this page again.
- Use the command palate to switch back to the text editor. It should load at the point where the static preview left off.

Issue 2:
- Open a markdown file as a static preview (`View - Reopen Editor With...` > `Markdown Preview (Experimental)` in command palette).
- Open the same file in another column, but in the regular text editor.
- Scroll both sides. Both sides should scroll along with one another.

This PR addresses #84520
